content/en/account_management/multi-factor_authentication.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Multi-Factor Authentication (MFA)
+kind: documentation
+---
+
+## Overview
+
+Multi-Factor Authentication (MFA), or Two-Factor Authentication (2FA) requires a user to present more than one type of verification to authenticate to a system. MFA defends against the majority of password-related attacks, including brute-force, credential stuffing, and password spraying.
+
+## Capabilities
+
+-   **MFA for native Datadog accounts**: MFA is available as an extra layer of security during login for accounts that log into Datadog directly using an email and password. Native email/password accounts are more vulnerable to attack than accounts maintained through an identity provider.
+-   **Opt-in MFA**: MFA is available for end users as an optional feature. Enable MFA at any time through your personal settings.
+-   **Authenticator apps**: Any authenticator app that supports time-based one-time password (TOTP) authentication can be used for MFA. Examples include Microsoft Authenticator, Google Authenticator, Authy, and Duo.
+
+## Limitations
+
+-   MFA is not available for accounts using Single Sign-On (SSO) only. To use MFA with SAML and Google Auth, configure it through your Identity Provider (IdP).
+-   MFA does not protect against all types of attacks. For example, if an attacker has access to your email, they may be able to turn off MFA and compromise your account.
+-   MFA supports at most one authenticator app.
+
+## Prerequisites
+
+To configure MFA for your account, log in using your email and password. Users who log in using SSO do not see MFA configuration options.
+
+## Configure MFA for your user account
+
+To find the [Password & Authentication page][1]:
+
+1. Navigate to **Personal Settings** from your account menu.
+1. Under **Security**, select **Password & Authentication**.
+
+The multi-factor authentication section lists any configured authenticator apps.
+
+1. Next to **Authenticator App**, select **Add**.
+1. Follow your authenticator app's documentation for instructions on adding a new QR code.
+1. Enter the latest code generated by your authenticator app into the prompt to confirm the device was set up correctly.
+1. Save a copy of the recovery codes in a secure location. The codes cannot be retrieved after setup is complete.
+
+## MFA recovery
+
+If you don't have access to your authenticator app, during the login process, you can use a recovery code instead of a one-time password. Each of the recovery codes can only be used once.
```

Comment:
```suggestion
If you don't have access to your authenticator app, during the login process you can use a recovery code instead of a one-time password. Each of the recovery codes can only be used once.
```